voter-dapp/src/ActiveRequests.js

```diff
@@ -63,6 +69,40 @@ function ActiveRequests({ votingAccount, votingGateway }) {
     ])
   );
 
+  const closedDialogIndex = -1;
+  const [dialogContentIndex, setDialogContentIndex] = useState(closedDialogIndex);
+  const handleClickExplain = index => {
+    setDialogContentIndex(index);
+  };
+
+  const handleClickClose = () => {
+    setDialogContentIndex(closedDialogIndex);
+  };
+
+  const proposals = useCacheCall(["Governor"], call => {
+    if (!initialFetchComplete) {
+      return null;
+    }
+    return pendingRequests.map(request => ({
+      proposal: isAdminRequest(hexToUtf8(request.identifier))
+        ? call("Governor", "getProposal", getAdminRequestId(hexToUtf8(request.identifier)))
+        : null
+    }));
+  });
+  const decodeRequestIndex = index => {
+    if (index === closedDialogIndex) {
+      return "";
+    }
+    const proposal = proposals[index].proposal;
+    let output =
+      hexToUtf8(pendingRequests[index].identifier) + " (" + proposal.transactions.length + " transaction(s))";
```

Comment:
did you forget to close these parentheses?

Reply:
No, I think this one is correctly closed.

Reply:
true